include/aspect/introspection.h

```diff
@@ -29,10 +29,265 @@
 
 #include <aspect/parameters.h>
 
+namespace dealii
+{
+  /**
+   * Represent a single variable of the deal.II finite element system
+   */
+  template <int dim>
+  struct Variable
+  {
+      /**
+       * Constructor for a variable.
+       *
+       * @param name A user-friendly and unique string representation.
+       *
+       * @param fe The FiniteElement class to use.
+       *
+       * @param multiplicity Number of copies of the @p fe to create.
+       *
+       * @param n_blocks Number of blocks this variable represents inside the
+       * linear system. A value of 0 will add the next variable to the current
+       * block, while 1 will put the next variable into a new block. A value that is
+       * equal to the number of components will create a block for each
+       * component.
```

Comment:
This may not matter because few users actually read this. But I don't understand this paragraph because I lack the background of what "blocks" refer to here, how they are used, and how they are used by this class and the classes above it.